components/BottomMenuBar.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, StyleSheet } from 'react-native';
+import { BottomTabBarProps } from '@react-navigation/bottom-tabs';
+import Icon from 'react-native-vector-icons/Ionicons';
+
+function getIconName(routeName: string) {
+  switch (routeName) {
+    case 'Schedule':
+      return 'calendar-outline';
+    case 'Recently Played':
+      return 'albums-outline';
+    case 'About':
+      return 'information-circle-outline';
+    case 'Home':
+      return 'home-outline';
+    default:
+      return 'ellipse-outline';
+  }
+}
+
+export default function BottomMenuBar({ state, navigation }: BottomTabBarProps) {
+  const activeIndex = state.index;
+
+  return (
+    <View style={styles.container}>
+      {state.routes.map((route, idx) => {
+        const focused = idx === activeIndex;
+        const iconName = getIconName(route.name);
+
+        return (
+          <TouchableOpacity
+            key={route.key}
+            style={styles.tab}
+            onPress={() => navigation.navigate(route.name)}
+            activeOpacity={0.8}
+          >
+            <Icon name={iconName} size={22} color={focused ? '#00D17A' : '#888'} />
+            <Text style={[styles.label, focused && styles.labelActive]}>{route.name}</Text>
```

Comment:
we'll also want to wrap this array (in the new-component-form of the buttons) in a useMemo (new objects to a prop can trigger a re-render, unless the component guards against)